File: Backend/crypt/crypt.py
#!/usr/bin/env python

import base64
import hashlib
from Crypto import Random
from Crypto.Cipher import AES

BS = 16
pad = lambda s: s + (BS - len(s) % BS) * chr(BS - len(s) % BS)
unpad = lambda s : s[0:-s[-1]]


class AESCipher:

    def __init__( self, key ):
        self.key = hashlib.sha256(key.encode('utf-8')).digest()

    def encrypt( self, raw ):
        raw = pad(raw)
        iv = Random.new().read( AES.block_size )
        cipher = AES.new( self.key, AES.MODE_CBC, iv )
        return base64.b64encode( iv + cipher.encrypt( raw ) )

    def decrypt( self, enc ):
        enc = base64.b64decode(enc)
        iv = enc[:16]
        cipher = AES.new(self.key, AES.MODE_CBC, iv )
        return unpad(cipher.decrypt( enc[16:] ))


cipher = AESCipher('#@protraderbotcriptografia#@')
encrypted = cipher.encrypt('r76Q8Q35KipGTmY4BRuQqx1LDp3fWFvMjGYK9avKZnvbXVAzitWNKupzUtoQ34Uj')
decrypted = cipher.decrypt(encrypted)
print(encrypted)

php = 'Xa/dagrtYtSwQUaHJtfV5V16DEJzJvZTyGJHxjKlgzT5SVuhSGSUqf6UneP/0vEsCDsU1QF8PpvwMe3vm1ReQlgBbBH+UovZ6veI+LgOvi0='

print (php)

if (php == encrypted):
    print (sucess)

File: Backend/crypt/crypt.php
<?php
        $plaintext = 'r76Q8Q35KipGTmY4BRuQqx1LDp3fWFvMjGYK9avKZnvbXVAzitWNKupzUtoQ34Uj';
        $password = '#@protraderbotcriptografia#@';
        $method = 'aes-256-cbc';

        // Must be exact 32 chars (256 bit)
        $password = substr(hash('sha256', $password, true), 0, 32);
        echo "Password:" . $password . "\n";

        // IV must be exact 16 chars (128 bit)
        $iv = chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0) . chr(0x0);

        // av3DYGLkwBsErphcyYp+imUW4QKs19hUnFyyYcXwURU=
        $encrypted = base64_encode(openssl_encrypt($plaintext, $method, $password, OPENSSL_RAW_DATA, $iv));

        // My secret message 1234
        $decrypted = openssl_decrypt(base64_decode($encrypted), $method, $password, OPENSSL_RAW_DATA, $iv);

        echo 'plaintext=' . $plaintext . "\n";
        echo 'cipher=' . $method . "\n";
        echo 'encrypted to: ' . $encrypted . "\n";
        echo 'decrypted to: ' . $decrypted . "\n\n";
    ?>
